auth/authpb/auth.proto

```diff
@@ -15,3 +15,10 @@ message User {
   bytes password = 2;
   int64 tombstone = 3;
 }
+
+// Role is a single entry in the bucket authRoles
+message Role {
+  int64 ID = 1;
```

Comment:
Yes, but there's no way to revoke deleted roles from entire users that are granted the roles. I think providing the way would be safer for auth management.